src/smexperiments/trial_component.py

```diff
@@ -92,8 +92,23 @@ def save(self):
         """Save the state of this TrialComponent to SageMaker."""
         return self._invoke_api(self._boto_update_method, self._boto_update_members)
 
-    def delete(self):
-        """Delete this TrialComponent from SageMaker."""
+    def delete(self, disassociate=None):
```

Comment:
Maybe call it "force_disassociate"?

Reply:
should we rename it? if we rename it, the default behavior will change(the delete method wont exist with this change).

Reply:
Oh I meant the parameter name.

Reply:
ya, thats better, I will rename it to force_disassociate